### ISSUE TITLE
Add support for wildcard exports

### ISSUE THREAD

**Issue by mseyfayi** [huozhi/bunchee#199]
Hi, I have a UI package with multi entries (e.g. button, input, ...) and want them to have separated bundles.
The solution with the bunchee is to use `pkg.exports`:

```JSON
"exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    },
    "./button": {
      "types": "./dist/button.d.ts",
      "import": "./dist/button.js"
    },
    "./input": {
      "types": "./dist/input.d.ts",
      "import": "./dist/input.js"
    },
    // ...
  }
```

But in this solution I need to put all of my components into `exports`, if I removed one I should remove it from `export`, and the same goes for adding.

The better solution would be to write my `pkg.exports` like this:

```JSON
"exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    },
    "./*": {
      "types": "./dist/*.d.ts",
      "import": "./dist/*.js"
    }
  }
```

But beloved bunchee is not supporting this. :(  
Would you consider adding this feature?

**Comment by huozhi** [comment]
Hey @mseyfayi this is def sth bunchee should support, thanks for bringing it up. Marked as a `feature request` and I'll work on it soon

**Comment by devjiwonchoi** [comment]
@huozhi I'm thinking of using `fs.readdir` or `glob` to get file names based on path. What do you think?

Also, could this be simply

```ts
"exports": {
  "./*": {
    "types": "./dist/*.d.ts",
    "import": "./dist/*.js"
  }
}
```

since `index.ts` is readable also?

**Comment by huozhi** [comment]
We've landed this feature in `3.7.x`, feel free to try it out! The usage is updated in the readme
